### ISSUE TITLE
Does KeyframeTracks parameter times, values have to be any type?

### ISSUE THREAD

**Issue by Suprhimp** [three-types/three-ts-types#294]
Hi, I faced the runtime error because of my keyframe value has Nan value Because I put the wrong type of the input to make keyframe.

In `THREE.VectorKeyframeTrack` the paramter `times` and `values` have to be number array.

First I made the code like this.
```
posKF = new THREE.VectorKeyframeTrack('.position',timesList, [camera.position], THREE.InterpolateSmooth);
```
But as you know I missed `.toArray()` function for camera position value to array of number not object.
```
posKF = new THREE.VectorKeyframeTrack('.position',timesList, [...camera.position.toArray()], THREE.InterpolateSmooth);
```

When I used the code with first way It compiled but I have to check `posKF` value because of not working well.

I logged `posKF` , it gives me this
![image](https://user-images.githubusercontent.com/73486185/207858507-a3c4a7b3-55e2-430e-8b68-03f6b7691758.png)

The values of keyframe has `Nan` value.

I think  `VectorKeyframeTrack`'s  parameter `times` and `values` have to use type `number[]` instead of `any[]`.

And so on `KeyframeTrack`'s parameter `times` and `values` have to use type `ArrayLike<number>` instead of `ArrayLike<any>` .

Does I'm thinking wrong? If it is not, I want to make PR for that.

Thanks for read :)

**Comment by 0b5vr** [comment]
Docs: https://threejs.org/docs/#api/en/animation/KeyframeTrack
Implementation: https://github.com/mrdoob/three.js/blob/309b00afb6dcbc5e6c58e72f10eaa8d2e8888c83/src/animation/KeyframeTrack.js#L20-L21
`AnimationUtils.convertArray`: https://github.com/mrdoob/three.js/blob/309b00afb6dcbc5e6c58e72f10eaa8d2e8888c83/src/animation/AnimationUtils.js#L19-L33

is it `ArrayLike<number>` ? Low confidence

**Comment by Suprhimp** [comment]
As document saids
```
times - an array of keyframe times, converted internally to a Float32Array
values - an array with the values related to the times array, converted internally to a Float32Array.
```

My opinion is `time` and `value` has to be number convert value to `Float32`

Shall I ask in three js github issue?

**Comment by 0b5vr** [comment]
Issue is too much for ask this imo. I just asked in Discord

**Comment by 0b5vr** [comment]
donmccurdy says that `ArrayLike<number>` looks correct.

https://discord.com/channels/685241246557667386/1055509846776102942/1055565437900947547

I believe `VectorKeyframeTrack` also receives `ArrayLike<number>`.

**Comment by Suprhimp** [comment]
Okay :) than Can I make the PR of this?

**Comment by 0b5vr** [comment]
yes, please!

**Comment by joshuaellis** [comment]
resolved by #306
